src/guiEngine.cpp

```diff
@@ -230,7 +230,7 @@ GUIEngine::GUIEngine(	irr::IrrlichtDevice* dev,
 /******************************************************************************/
 bool GUIEngine::loadMainMenuScript()
 {
-	// Try custom menu script (main_menu_path)
+	// Set main_menu_path
```

Comment:
How does this "set" the main_menu_path? It only reads it. I agree though, the old comment is misleading, so it should be changed.